app/app.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div class="dark:bg-customBlack dark:text-dark-white text-customBlack h-screen flex flex-col items-center">
+    <div class="h-full w-full flex flex-col pt-10 px-4 text-base max-w-screen-lg" style="max-height: calc(100% - 39px);">
+      <NuxtPage />
+    </div>
+    <div class="w-full border-t text-center py-2 px-6 text-sm bg-slate-100 dark:bg-dark-lightgray">
+      <span class="dark:text-neutral-500 text-neutral-600">made with </span>
+      <a href="https://github.com/sidestream-tech/sidebase" class="text-SideBase-green" target="_blank">
+        SideBase
+      </a>
+    </div>
+  </div>
+</template>
+
+<style>
```

Comment:
please always use `style sacoped` where possible, see https://vue-loader.vuejs.org/guide/scoped-css.html - this way style does not pollute the global website.

please fix everywhere

Reply:
I used scoped style in the other components but wanted this style to be a global style.
For this, if this sounds better I can also make a separate CSS file for global styles.